applications/nrf5340_audio/src/bluetooth/ble_audio_services.h

```diff
@@ -10,23 +10,11 @@
 #include <zephyr/bluetooth/conn.h>
 
 /**
- * @brief Discover VCS and included services
+ * @brief  Callback for changing stream state.
  *
- * @param conn Pointer for peer connection information
- * @param channel_num The number of the remote device
- *
- * This will start a GATT discovery and setup handles and subscriptions for
- * VCS and included services.
- * This shall be called once before any other actions related with VCS.
- *
- * @return 0 for success, error otherwise.
- */
-int ble_vcs_discover(struct bt_conn *conn, uint8_t channel_num);
-
-/**
- * @brief Initialize the Volume Control Service client
+ * @param  play Differentiate between play command and pause command.
  */
-int ble_vcs_client_init(void);
+typedef void (*ble_mcs_play_pause_cb)(bool play);
```

Comment:
Is `typedef` required?

Reply:
I believe so for defining a callback, that's what we do in le_audio.h:127, ble_core.h:14, power_module.h:17 etc